packages/core/src/components/ModalNew/Modal/Modal.tsx

```diff
@@ -78,24 +79,26 @@ const Modal = forwardRef(
             onClick={onBackdropClick}
             aria-hidden
           />
-          <div
-            ref={ref}
-            className={cx(styles.modal, getStyle(styles, camelCase("size-" + size)), className)}
-            id={id}
-            data-testid={dataTestId || getTestId(ComponentDefaultTestId.MODAL_NEXT, id)}
-            role="dialog"
-            aria-modal
-            aria-labelledby={titleId || undefined}
-            aria-describedby={descriptionId || undefined}
-          >
-            <ModalTopActions
-              renderAction={renderHeaderAction}
-              color={closeButtonTheme}
-              closeButtonAriaLabel={closeButtonAriaLabel}
-              onClose={onClose}
-            />
-            {children}
-          </div>
+          <FocusLock returnFocus>
```

Comment:
We need to use it also in Dialogs in the future

Reply:
yep, we need to maybe implement it in all the system modals and old modal, and therefore all the locks would use the same lib and can live in harmony, until all would migrate to the new modal